multi-node-test.sh

```diff
@@ -148,114 +107,106 @@ df -h" 2>&1 | dshbak -c; then
     exit 1
 fi
 
+if ! ssh -i ci_key jenkins@"${nodes[0]}" "set -x
+set -e
+sudo mkdir -p \$CART_BASE
+sudo mount -t nfs \$HOSTNAME:\$PWD \$CART_BASE"
+fi
```

Comment:
(lint) Expected 'then'. [SC1050]
(lint) Expected "#". Fix any mentioned problems and try again. [SC1072]